spec/config_manager/http_project_config_manager_spec.rb

```diff
@@ -24,7 +24,7 @@
 require 'optimizely/logger'
 describe Optimizely::HTTPProjectConfigManager do
   let(:config_body_JSON) { OptimizelySpec::VALID_CONFIG_BODY_JSON }
-  let(:error_handler) { Optimizely::NoOpErrorHandler.new }
+  let(:error_handler) { Optimizely::RaiseErrorHandler.new }
```

Comment:
What are the implications of changing the default behavior from no op error handler to one that raises an error?  Couldn't this update catch people off guard?

Reply:
This doesn't change the default Error Handler for the cient, if the client doesn't pass a custom error handler, still a NoOpErrorHandler will be used.  We are only using RaiseErrorHandler in unit tests so that we catch up any error that we throw without passing required params.